sql/src/main/java/io/crate/action/sql/SQLOperations.java

```diff
@@ -147,17 +146,38 @@ public void simpleQuery(String query,
                 query = query.substring(0, query.length() - 1);
             }
 
-            parse("", query, Collections.<DataType>emptyList());
-            bind("", "", Collections.emptyList());
-            List<Field> fields = describe('S', "");
+            Statement statement = SqlParser.createStatement(query);
+            Analysis analysis = analyzer.analyze(statement, new ParameterContext(EMPTY_ARGS, EMPTY_BULK_ARGS, defaultSchema));
+            Plan plan = planner.plan(analysis, UUID.randomUUID(), 0, 0);
 
-            ResultReceiver resultReceiver = resultReceiverFactory.apply(fields);
-            execute("", 0, resultReceiver);
-            sync(doneListener);
+            ResultReceiver resultReceiver;
+            if (analysis.rootRelation() == null) {
```

Comment:
Don't think that it adds readability here - it wouldn't fit into one line either way